pkg/vault/test/foundry/BalancerPoolTokenTest.t.sol

```diff
@@ -312,4 +316,22 @@ contract BalancerPoolTokenTest is BaseVaultTest {
     function testSupportsIERC165() public view {
         assertTrue(poolToken.supportsInterface(type(IERC165).interfaceId), "IERC165 not supported");
     }
+
+    function testGetVault() public {
+        assertEq(address(poolToken.getVault()), address(vault), "Vault is wrong");
+    }
+
+    function testGetRate() public {
+        // Init pool, so it has a BPT supply and rate can be calculated.
+        vm.startPrank(lp);
+        IERC20[] memory tokens = vault.getPoolTokens(address(poolToken));
+        router.initialize(address(poolToken), tokens, [poolInitAmount, poolInitAmount].toMemoryArray(), 0, false, "");
+        vm.stopPrank();
+
+        uint256[] memory liveBalancesScaled18 = vault.getCurrentLiveBalances(address(poolToken));
+        uint256 invariant = IBasePool(address(poolToken)).computeInvariant(liveBalancesScaled18);
+        uint256 bptRate = invariant.divDown(poolToken.totalSupply());
+
+        assertEq(poolToken.getRate(), bptRate, "BPT rate is wrong");
```

Comment:
The rate should be 1 here, right after initializing. Consider also adding:

`assertEq(bptRate, FixedPoint.ONE);`